framework/material/ActionableList.kt

```diff
@@ -71,20 +71,29 @@ object ActionableList {
 
         Box(modifier) {
             Row(Modifier.verticalScroll(scrollState)) {
-                if (buttonSide == Side.LEFT) {
-                    ActionColumn(items, itemHeight, buttonFn)
+                if (buttonsSide == Side.LEFT) {
+                    ActionColumn(items, itemHeight, buttonsFn)
                     Separator()
                 }
                 ItemColumn(Modifier.weight(1f), items, itemHeight)
-                if (buttonSide == Side.RIGHT) {
+                if (buttonsSide == Side.RIGHT) {
                     Separator()
-                    ActionColumn(items, itemHeight, buttonFn, scrollState)
+                    ActionColumn(items, itemHeight, buttonsFn, scrollState)
                 }
             }
             Scrollbar.Vertical(rememberScrollbarAdapter(scrollState), Modifier.align(Alignment.CenterEnd))
         }
     }
 
+    @Composable
+    fun <T: Any> SingleButtonLayout(
```

Comment:
this name insn't symmetric with the `Layout` name you've chosen for the multi-button version. I would call both `ButtonLayout` and have two overloaded versions that have different argument signatures. What do you think?

Reply:
As discussed.